packages/code-du-travail-frontend/src/layout/Footer.js

```diff
@@ -148,6 +148,55 @@ const Footer = () => (
           </Links>
         </nav>
       </NavSection>
+      <GovernmentSection>
+        <StyledGovLink
+          href={"https://travail-emploi.gouv.fr"}
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          travail-emploi.gouv.fr
+        </StyledGovLink>
+        <Separator>|</Separator>
```

Comment:
I would add an aria-hidden  / user-select: none;to Separator element

Reply:
added